Src/AutoMoqUnitTest/AutoMoqUnitTest.csproj

```diff
@@ -12,6 +12,9 @@
     <!--  Suppress warning about invalid dependency version in Castle.Core.
           That is Moq dependency and we cannot fix that somehow. -->
     <NoWarn>$(NoWarn);NU1603</NoWarn>
+
+    <!-- In the latest versions of Moq (consumed by .Net Core) they fixed issue with out parameters for delegates. -->
```

Comment:
perhaps "they fixed an issue" or even remove the "they":

"The latest versions of Moq (consumed by .Net Core) fixed an issue with out parameters for delegates."

Reply:
Rephrased 😉